crowbar_framework/lib/crowbar/backup/restore.rb

```diff
@@ -285,8 +285,9 @@ def restore_database
       end
 
       def proposal?(file)
-        !file.basename.to_s.match(/(_network\.json$)|(^template-(.*).json$)|(^queue\.json$)/) && \
-          file.to_s =~ /knife\/databags\/crowbar/
+        !file.basename.to_s.match(
+          /(_network\.json$)|(^template-(.*).json$)|(^queue\.json$)|(^swift\.json$)/
+        ) && file.to_s =~ /knife\/databags\/crowbar/
```

Comment:
Coming back to this after some more thoughts... I don't have a Cloud 5 data dump here, so I cannot check, but I think all proposals are actually starting with `bc-`. I feel that would be the right check, actually.

Is there a data dump somewhere to look at for this?